app/GraphQL/Inventory/Mutations/Status/StatusMutation.php

```diff
@@ -67,4 +69,24 @@ public function delete(mixed $rootValue, array $request): bool
 
         return $status->delete();
     }
+
+    /**
+     * update.
+     */
+    public function updateStatusTranslation(mixed $root, array $req): StatusModel
+    {
+        $company = auth()->user()->getCurrentCompany();
+        $language = Languages::getByCode($req['code']);
```

Comment:
This code is repeated in other mutation, maybe you need to create a parent class or trait